lib/client/customer.rb

```diff
@@ -12,7 +12,12 @@ def get(id, params = {})
           return response.body['data'][0]
         rescue Faraday::Error::ClientError => e
           raise BigcommerceApiError.new(
-            'get order', "Failed to get order #{id}", { order_id: id }, e
+            500,
+            'get order customer',
+            "Failed to get order customer #{id}",
+            id,
+            { customer_id: id },
+            e
```

Comment:
@Jhalter5Stones The `status` value here should be coming from the faraday response, and we should be including any errors returned from BigCommerce (much like the product agent is doing.

Additionally, the process that generates error events probably needs to be updated to handle this modified error structure.

Reply:
@mrogers-5s I think I resolved this but let me know what you think